job.c

```diff
@@ -318,8 +318,9 @@ create_mail(cl_t * line, char *subject, char *content_type, char *encoding,
     hostname[0] = '\0';
 #endif                          /* HAVE_GETHOSTNAME */
 
-    /* write mail header */
-    fprintf(mailf, "From: %s%s (fcron)\n", mailfrom, hostname_from);
+    /* write mail header. displayname comes from fcron.conf */
+    fprintf(mailf, "From: %s %s%s\n", displayname != NULL ? displayname : "",
+            mailfrom, hostname_from);
```

Comment:
@CharlieReitzel, you're indeed correct about formatting, thanks. I'll include that.
@yo8192, I'll dig a bit more about compiling it in Gentoo -- had issues with building the docs, otherwise it looks OK. BTW, I don't see (unit) tests in the source code, are there any?

Reply:
> If a display name is used, the actual email address be wrapped in angle brackets. Also, if the display name contains spaces, it should be quoted. E.g. `"Joe Bob" <jb@example.com>` vs `Joebob <jb@example.com>` vs `jb@example.com`.

Back on this. As per [RFC5322 sec. A.1.2](https://datatracker.ietf.org/doc/html/rfc5322#appendix-A.1.2), quoting is only needed if the display name has the following characters: period, semicolon and double-quote. So, I wonder if we wouldn't better off using some library function to correctly deal with it. Any idea? Meanwhile looking around....

Reply:
generally speaking I'd rather limit the number of dependencies fcron has. But if it does make sense to use one (e.g. the code wouldn't be obvious), then please make it optional using autoconf (the `./configure` script).